app/src/main/res/layout/city_layout.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activities.MainActivity"
+    android:background="@color/light_green_color">
+
+    <TextView
+        android:id="@+id/welcome_text_view"
```

Comment:
pls add to the name the <what> and <where> section
https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Reply:
Done!